### PR TITLE
Feature/product page

### PR DESCRIPTION
**Pull request by vytautasBrundza**

### CODE REVIEW

File: product.php [hunk 1]
```diff
@@ -4,34 +4,52 @@
...
+        <div class="grid">
+            <div class="grid__item one-third palm-one-whole">
+                <img src="http://placekitten.com/g/550/550" alt="">
+            </div><!--
+         --><div class="grid__item two-thirds palm-one-whole">
+                <h1>Product Name</h1>
+                <p>
+                    Duis vehicula laoreet lacus sit amet luctus. Donec tempor pulvinar nunc nec rhoncus. Nullam pulvinar placerat nisi ac viverra. Nullam egestas, lacus quis fermentum rhoncus, magna enim consectetur nunc, in fringilla leo felis sit amet sem. Nunc nisi nunc, vehicula mattis justo a, tempor convallis lectus. Morbi ac ante tortor. Fusce vitae nunc ut nunc vehicula luctus id nec ex.
+                </p>
+                <p><strong># <span class="js-percent1"></span>%</strong></p>
```
Comment (VerifiedMarfat) [inline product.php]: What does the class 'js-percent1' do? If we dont use it, we should remove it.

File: product.php [hunk 2]
```diff
@@ -4,34 +4,52 @@
...
+            <div class="push--bottom">
+                <span class="push--sides">Price</span>
+                <span class="+lined">$$$</span>
+            </div>
+            <form action="" class="text--right">
+                <input type="submit" value="Buy" name="checkout" class="btn one-third lap-two-thirds palm-one-whole">
             </form>
+            <div class="text--right">
+                <div class="social one-third lap-two-thirds palm-one-whole product-share-cnt">
+                    <ul class="nav nav--block">
```
Comment (VerifiedMarfat) [inline product.php:42]: The class 'product-share-cnt' doesnt follow BEM naming convention. Please change. For example, its a share button, so something like 'btn--share' is more appropriate.

File: src/sass/objects/_product.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.product-share-cnt {
```
Comment (VerifiedMarfat) [inline src/sass/objects/_product.scss]: If this is the only styling that this class adds, we should consider creating a helper class instead.

File: product.php (hunk 2 above)
Comment (vytautasBrundza) [inline product.php:42]: I thought of it as share container, but can remove the cnt.

**Comment by vytautasBrundza** [comment]
Made changes according to comments. I keep forgetting that you output everything in PHP. Created those js classes for content replacement with ajax :D They are removed now including the spans

**Pull request merged**